tsconfig.json

```diff
@@ -2,14 +2,14 @@
   "extends": "@strapi/typescript-utils/tsconfigs/server",
   "compilerOptions": {
     "outDir": "dist",
-    "rootDir": "."
+    "rootDir": ".",
+    "allowJs": true,
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true
   },
-  "include": [
-    "./",
-    "./**/*.ts",
-    "./**/*.js",
-    "src/**/*.json"
-  ],
+  "include": ["./", "./**/*.ts", "./**/*.js", "src/**/*.json"],
```

Comment:
**question(non-blocking):** I find it easier to parse inline array definitions like this in the original list style, instead of squishing them all onto one line.  Is it possible to set the formatter to use the list style for inline arrays with 3+ entries?

Reply:
Unfortunately, no, it's based on width and it'll only break it over multiple lines if it's over 80 characters wide...